src/main/java/webserver/Response.java

```diff
@@ -0,0 +1,113 @@
+package webserver;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Response {
+    private static final String CRLF = "\r\n";
+    private static Map<Integer, String> responseStatus = buildResponseCodes();
+    private PrintWriter out;
+    private String path;
+    private int statusCode;
+    private String body = "";
+    private String requestMethod;
+
+
+    public Response(PrintWriter out, String path, String requestMethod) {
+        this.out = out;
+        this.path = path;
+        this.requestMethod = requestMethod;
+    }
+
+    private static Map<Integer, String> buildResponseCodes() {
+        Map<Integer, String> responses = new HashMap<Integer, String>();
+        responses.put(200, "OK");
+        responses.put(404, "Not Found"); 
+        return responses;
+    }
+
+    public void setupDataToBeSent() throws IOException {
+        Path parsedPath = createFilePath(createPathString(path));
+        setStatusCode(parsedPath);
+        if (requestMethod.equals("GET")){
+            getHTML(parsedPath);
+        }
+    }
+
+    public void send() throws IOException {
+        out.print(createInitialResponseLine(statusCode));
+        out.print(CRLF);
+        out.print("Content-Length: " + body.length());
+        out.print(CRLF);
+        out.print("Content-Type: text/html; charset=utf˝-8");
+        out.print(CRLF);
```

Comment:
I'm thinking that there's probably a better way to organize this so that if you have a hashmap of headers, it'll know how to convert that into this. What do you think?